nowcasting_dataset/data_sources/fake.py

```diff
@@ -238,6 +238,7 @@ def create_gsp_pv_dataset(
 
     data["x_coords"] = x_coords
     data["y_coords"] = y_coords
+    data["pv_system_row_number"] = data["id"] + 1000
```

Comment:
Why is 1000 added here?

Reply:
just wanted it a bit different that the row number, could generate random numbers instead

Reply:
Ah okay, could we add a comment explaining it then?